tools.go

```diff
@@ -1,8 +1,11 @@
 // (c) 2025, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
 
-package subnetevm
+//go:build tools
+
+package coreth
 
 import (
+	_ "github.com/fjl/gencodec"
```

Comment:
why is this added?

Reply:
To be able to pin the gencodec version in go.mod without having go mod tidy throwing it out 😄